tests/core/data_layer/test_data_rpc.py

```diff
@@ -116,7 +116,7 @@ async def farm_block_with_spend(
 
 
 def check_mempool_spend_count(full_node_api: FullNodeSimulator, num_of_spends: int) -> bool:
```

Comment:
Hey, while you're in here, could you change `num_of_spends` to something like `num_of_fee_rate_levels` ?

Reply:
but with my change this value now refers to the number of spends in the mempool. the public interface to the mempool class no longer gives you access to the `sorted_spends` list